src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -1672,7 +1671,38 @@ public void testCheckoutSansLocalBranchExtension() throws Exception {
        assertEquals("GIT_BRANCH", "origin/master", getEnvVars(project).get(GitSCM.GIT_BRANCH));
        assertEquals("GIT_LOCAL_BRANCH", null, getEnvVars(project).get(GitSCM.GIT_LOCAL_BRANCH));
     }
+     /**
```

Comment:
I generally prefer a blank line after the end of a method, if you're willing to add them here.

Reply:
added to make it more consistent